### ISSUE TITLE
User story - Activity feed

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#20]
### If applicable, add a related Epic.

Epic: #14 

### User Story

As a **user**, I can **access an activity feed** so that I can **see recent updates made by connections or people I follow**.

### Acceptance Criteria

- The user should have an activity feed view they can access
- User should be able to sort the activity feed
- User should be able to filter the activity feed
- User should be able to exclude certain posts/activity from their feed

### Please ensure you've completed these tasks before marking the issue as done.

- [ ] Reviewed the user story for clarity and completeness.
- [ ] Identified and linked related issues or epics.
- [ ] Created or updated the necessary documentation.
- [ ] Written unit and/or integration tests for functionality.
- [ ] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [ ] Updated the issue with any relevant notes or feedback (to date).
- [ ] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [ ]
